unicon_runner/executor/templates/run_unsafe.sh.jinja

```diff
@@ -12,10 +12,10 @@ measure_elapsed_time() {
     local command="$1"
     local output_file="$2"
 
-    local start_time_ns=$(date +%s%N)
+    local start_time_ns=$(date +%s%N | sed 's/N$//')
```

Comment:
i am assuming this is support for macOS?

if so, macOS `date` command does not actually support nanosecond precision so the "N" at the back is treated literally (aka it just gets appended at the back as a suffix)

Reply:
i forgot to mention that this timer is not exactly platform agnostic since it relies on linux `date` command

Reply:
is that why it breaks locally for me - do you forsee this change breaking other OSes? 👀

Reply:
it shouldn't affect our main deployment target being the soc cluster so i think its fine to leave it in if we want it to work on mac - the only thing to note is that it will not be accurate